test/autoscaling-hpa-test-values.yaml

```diff
@@ -5,4 +5,4 @@ alfresco-repository:
     maxReplicas: 2
     behavior:
       scaleUp:
-        stabilizationWindowSeconds: 10
+        stabilizationWindowSeconds: 0
```

Comment:
This is to make sure we dont delay the scaling process?

Reply:
yes, seems more sensibile for testing even if...